src/app/models/config/nodes.ts

```diff
@@ -1,3 +1,15 @@
+function getTitleLevel(title: Element): number {
+  let parent = title.parentNode;
+  let level = 1;
+  while (parent.nodeName !== 'body') {
```

Comment:
Might be wise to consider also adding a check to bail out of the loop if parent or parent.nodeName are underfined/null/etc.